.packit.yaml

```diff
@@ -1,31 +1,121 @@
 packages:
   atuin:
-    specfile_path: rust-atuin.spec
-    files_to_sync:
-      - rust-atuin.spec
-      - .packit.yaml
+    paths: [ atuin ]
+    specfile_path: atuin.spec
+    downstream_package_name: atuin
     upstream_package_name: atuin
-    downstream_package_name: rust-atuin
+    actions: &Actions-Package-rust2rpm-workspace
+      # Note: in upstream, the License has to be fixed manually after rust2rpm is run
```

Comment:
For my own clarification: does "upstream" here refer to this repo, when generating the spec file for new patch releases?

Reply:
I know I just wrote that sentence one week ago, but I forgot. But the gist of it is that the binary pulls in all other rust crates as static libraries so its license is the combination of all of them.